django-stubs/views/generic/edit.pyi

```diff
@@ -8,7 +8,7 @@ from django.utils.datastructures import _ListOrTuple
 from django.views.generic.base import ContextMixin, TemplateResponseMixin, View
 from django.views.generic.detail import BaseDetailView, SingleObjectMixin, SingleObjectTemplateResponseMixin
 
-_FormT = TypeVar("_FormT", bound=BaseForm)
+_FormT = TypeVar("_FormT", bound=BaseForm, default=BaseForm)
```

Comment:
question: why do we need a default here?

Reply:
Maybe we should recommande the --disallow-any-generics mypy flag somewhere in the docs

Reply:
Let's just require it :)

Reply:
> Let's just require the generic here :)

Sorry, not sure I follow what you expect me to do here. You prefer to skip the default ?